test/basic-querying.test.js

```diff
@@ -127,7 +127,23 @@ describe('basic-querying', function() {
 
   describe('find', function() {
 
-    before(seed);
+    before(function() {
```

Comment:
Note that your implementation has another problem - because your `setupDelayingLoadedHook` does not accept callback arg, mocha will think it's a sync function and won't wait for completion. As a result, some of the tests may run before the delaying observer is set up. Also you